src/gestures/drag/VisualElementDragControls.ts

```diff
@@ -83,19 +83,14 @@ export class VisualElementDragControls {
         originEvent: AnyPointerEvent,
         { snapToCursor = false }: DragControlOptions = {}
     ) {
-        let initialPoint: Point
-
         const onSessionStart = (event: AnyPointerEvent) => {
             // Stop any animations on both axis values immediately. This allows the user to throw and catch
             // the component.
             this.stopAnimation()
 
-            /**
-             * Save the initial point. We'll use this to calculate the pointer's position rather
-             * than the one we receive when the gesture actually starts. By then, the pointer will
-             * have already moved, and the perception will be of the pointer "slipping" across the element
-             */
-            initialPoint = getViewportPointFromEvent(event).point
+            if (snapToCursor) {
+                this.snapToCursor(extractEventInfo(event, "page").point)
+            }
```

Comment:
It probably doesn't make much of a difference, but should we snap only after the drag actually starts (in `onStart`)?

Reply:
It was this way before but also the feel of it is quite different, a bit like how we stop existing animations on mouse down not mouse move